src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 40 * days;
+
+  if (days >= 7) {
+    rentalCost = rentalCost - 50;
+  } else if (days >= 3) {
```

Comment:
Consider removing the `else` keyword as it is unnecessary after an `if` block that contains a `return` statement. This will make the code cleaner and is in line with checklist items 4 and 5.